ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/documentation/devices/host1.md

```diff
@@ -2,3 +2,136 @@
 
 ## Table of Contents
 
+- [Interfaces](#interfaces)
+  - [Ethernet Interfaces](#ethernet-interfaces)
+  - [Port-Channel Interfaces](#port-channel-interfaces)
+
+## Interfaces
+
+### Ethernet Interfaces
+
+#### Ethernet Interfaces Summary
+
+##### L2
+
+| Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
+| --------- | ----------- | ---- | ----- | ----------- | ----------- | ------------- |
+| Ethernet1 | - | access | 100 | - | - | - |
+| Ethernet2 | - | trunk | 110 | 10 | ['group1', 'group2'] | - |
```

Comment:
```suggestion
| Ethernet2 | - | trunk | 110 | 10 | 'group1', 'group2' | - |
```

Reply:
This is generated from template, will update

Reply:
Fixed